modules/kernel/src/org/apache/axis2/context/ConfigurationContextFactory.java

```diff
@@ -155,6 +155,13 @@ private static void setContextPaths(AxisConfiguration axisConfig,
             configContext.setServicePath(Constants.DEFAULT_SERVICES_PATH);
         }
 
+        // Checking for context path validation
+        Parameter servicePathValidation = axisConfig.getParameter(Constants.PARAM_SERVICE_PATH_VALIDATION);
+        if (servicePathValidation != null) {
+            boolean spathValidation = Boolean.parseBoolean(((String) servicePathValidation.getValue()).trim());
```

Comment:
can there be a parser error if value is not a boolean?

Reply:
no, it will return true for "true" ( equal ignore case ), for all other inputs it return false